client/mm/exchange_adaptor.go

```diff
@@ -2986,6 +3066,35 @@ func (u *unifiedExchangeAdaptor) transfer(dist *distribution, currEpoch uint64)
 	return true, nil
 }
 
+// tryTransfers attempts to optimize the asset distribution by moving funds
+// between the DEX and the CEX. If the distribution is already optimal, no
+// transfers are made. If the distribution is not optimal, the bot will first
+// attempt to use available funds to rebalance the distribution. If this is
+// not sufficient, an actual deposit or withdrawal will be done.
+func (u *unifiedExchangeAdaptor) tryTransfers(currEpoch uint64, distribution func(bool) (*distribution, error)) (actionTaken bool, err error) {
+	dist, err := distribution(false)
+	if err != nil {
+		return false, fmt.Errorf("distribution calculation error: %w", err)
+	}
+
+	baseInv, quoteInv := dist.baseInv, dist.quoteInv
+	if baseInv.toDeposit+baseInv.toWithdraw+quoteInv.toDeposit+quoteInv.toWithdraw == 0 {
+		return false, nil
+	}
+
+	complete := u.tryInternalTransfers(dist)
```

Comment:
Should internal transfers be a configurable option? If so, global or per bot or both?

Reply:
I think it should be default if external transfers are enabled.. I guess internal transfers could be configured separately as well, but I'm not sure why someone wouldn't want to enable them.

Reply:
I'll follow up with some UI changes for this. Currently if the `AutoRebalanceCfg` is nil, no rebalance will be attempted. If the min transfer amounts are 0, only the internal transfers will be attempted.